repositories/user.repository.js

```diff
@@ -0,0 +1,31 @@
+var mongoose = require('mongoose');
+
+var UserSchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true
+    },
+    password: {
```

Comment:
No need for changes to password for now, but just to note that we'll need to do some encryption on password.

Reply:
noted